charmcraft/templates/init/src/charm.py.j2

```diff
@@ -25,12 +25,16 @@ class {{ class_name }}(CharmBase):
         self._stored.set_default(things=[])
 
     def _on_config_changed(self, _):
+        # Note: you need to uncomment the example in the config.yaml file for this to work (ensure
```

Comment:
This is a tough call, perhaps it might be better to comment the code block entirely so it is a noop?

Reply:
As we just talked, I'll leave this so project tests work fine with just the YAML files "restored".